src/constants.js

```diff
@@ -19,6 +19,8 @@ export const ImageTypeLabels = {
   "edge-raw-image": "RHEL for Edge Raw Image (.raw.xz)",
   "image-installer": "RHEL Installer (.iso)",
   "edge-simplified-installer": "RHEL for Edge Simplified Installer (.iso)",
+  "edge-ami": "RHEL for Edge AMI (.raw)",
+  "edge-vsphere": "RHEL for Edge VMware VSphere (.vmdk)",
```

Comment:
Looking at https://www.vmware.com/products/cloud-infrastructure/vsphere the capitalisation should probably look like this.

```suggestion
  "edge-vsphere": "RHEL for Edge VMware vSphere (.vmdk)",
```

Reply:
Thanks!! Good catch

Reply:
It's capitalised incorrectly in other places, so I've fixed those too.

Reply:
Excellent, thanks!